src/EthersSafe.ts

```diff
@@ -10,31 +10,89 @@ class EthersSafe implements Safe {
   #ethers: any
   #signer: Wallet
 
+  /**
+   * Creates an instance of the Safe Core SDK.
+   *
+   * @param ethers - Ethers v5 library
+   * @param signer - Ethers signer
+   * @param safeAddress - The address of the Safe account to use
+   * @returns The Safe Core SDK instance
```

Comment:
shouldn't return also include a type? https://jsdoc.app/tags-returns.html

btw I think typescript supports jsdoc and can generate types from it, so maybe we don't need to duplicate them: https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#param-and-returns